mqtt/build/linux/cross-test.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
Do you think it is possible to combine test.sh and cross-test.sh into one?

Reply:
Yes, good point.